src/scenes/Partners/Detail/Tabs/Projects/PartnerDetailProjects.tsx

```diff
@@ -61,3 +68,22 @@ export const PartnerDetailProjects = () => {
     </TabPanelContent>
   );
 };
+
+const PartnershipColumns: GridColDef[] = [
+  {
+    field: 'partnerships.types',
+    ...multiSelectColumn(PartnerTypeList, PartnerTypeLabels),
+    headerName: 'Partnership Type',
+    width: 160,
+    serverFilter: ({ value }) => ({
+      partnerships: { types: value },
+    }),
+    valueGetter: (_, { partnership }) => partnership.types.value,
+  },
+];
```

Comment:
@CarsonF 
We tried to specifically type this to the correct fragment like you said to do but we were unable to get it working.  It says that the `ProjectDataGridRowFragment` and the `PartnerProjectDataGridRowFragment` are incompatible.  We tried several things to get past this without success.  I am sure you will be able to figure this out.  Maybe you, @atGit2021 and I can zoom about this tomorrow.